src/script/script.cpp

```diff
@@ -5,18 +5,11 @@
 
 #include "script.h"
 
+#include "script/interpreter.h"
```

Comment:
Would it be enough to move the opcode constants from script/interpreter to script/script in this PR to avoid this new include?
I haven't tried to compile that but I think that should be enough.

About IsPayToPublicKeyHash, do you mean ValueString shouldn't be a method of CScript ?
Otherwise, since CScript::ValueString uses IsPayToPublicKeyHash, it has to be in the same level or below, but not above.
If I have to suggest another place, I think I would go with script/standard, where other "static analysis" things are.